aptos-move/block-executor/src/executor.rs

```diff
@@ -714,6 +787,32 @@ where
         aggregator_v1_delta_writes
     }
 
+    /// Can be called (at most) once after transaction is committed to internally include
```

Comment:
This comment is very confusing. I see it called many times; also called after transaction is committed (which transaction - it's not an argument). What does `internally include` mean in this context?

Reply:
it's already outdated :) just deleted completely, the method just combines now and the name / API should be self-descriptive.